tests/README.md

```diff
@@ -5,58 +5,73 @@ contracts of Firecracker.
 
 ## Running
 
-To run all tests:
+The testing system is built around [pytest](https://docs.pytest.org/en/latest/).
+Our `tools/devtool` script is a convenience wrapper which automatically
+downloads necessary test artifacts from S3, before invoking pytest inside a
+docker container. For detailed help on usage, see `tools/devtool help`.
+
+To run all available tests that would also run as part of our PR CI (e.g.
+excluding tests marked with `pytest.mark.nonci`):
 
 ```sh
-tools/devtool test
+tools/devtool -y test
 ```
 
-This will download test microvm images from the default test resource S3 bucket
-and run all available tests.
-
-To run tests from specific directories and/or files:
+To run only tests from specific directories and/or files:
 
 ```sh
-tools/devtool test -- <test_dir_or_file_path>...
+./tools/devtool -y test -- integration_tests/performance/test_boottime.py
```

Comment:
Can we unify the devtool commands into either `./tools/devtool` or `tools/devtool` in this doc?

Reply:
oh, yeah, sure :)

Reply:
Done, it should all just be `tools/devtool` now